<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>offer_editor_dialog</class>
 <widget class="QDialog" name="offer_editor_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Offer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbItem">
     <property name="text">
      <string>Item:</string>
     </property>
     <property name="buddy">
      <cstring>cbItem</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="btItemNew">
     <property name="text">
      <string>+</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>417</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lbType">
     <property name="text">
      <string>Offer type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="localized_spin_box" name="sbPricePiece">
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="btItemEdit">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="document-properties">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="btItemDel">
     <property name="text">
      <string>-</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff/>
      </iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lbBalance">
     <property name="text">
      <string>Your Balance:</string>
     </property>
     <property name="buddy">
      <cstring>sbBalance</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lbExpires">
     <property name="text">
      <string>Expires:</string>
     </property>
     <property name="buddy">
      <cstring>dtExpires</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="4">
    <widget class="localized_spin_box" name="sbTotal">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="value">
      <number>21</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="cbType">
     <item>
      <property name="text">
       <string>Buy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sell</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lbTotalPrice">
     <property name="text">
      <string>Total Price:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>sbTotalPrice</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="4">
    <widget class="QLabel" name="lbAmount">
     <property name="text">
      <string>Amount:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>sbAmount</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="lbFee">
     <property name="text">
      <string>Fee:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>sbFee</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lbPricePiece">
     <property name="text">
      <string>Piece Price:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>sbPricePiece</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="4">
    <widget class="localized_spin_box" name="sbFee">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="7">
    <widget class="QComboBox" name="cbItem">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="lbTotal">
     <property name="text">
      <string>Total Price Including Fees:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>sbTotal</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="4">
    <widget class="localized_spin_box" name="sbAmount">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="localized_spin_box" name="sbTotalPrice">
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="localized_spin_box" name="sbBalance">
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> gp</string>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDateTimeEdit" name="dtExpires">
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd HH:mm:ss</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>localized_spin_box</class>
   <extends>QSpinBox</extends>
   <header>localized_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbItem</tabstop>
  <tabstop>btItemNew</tabstop>
  <tabstop>btItemDel</tabstop>
  <tabstop>btItemEdit</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>sbTotalPrice</tabstop>
  <tabstop>sbPricePiece</tabstop>
  <tabstop>sbAmount</tabstop>
  <tabstop>sbBalance</tabstop>
  <tabstop>sbFee</tabstop>
  <tabstop>sbTotal</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>offer_editor_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>offer_editor_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
